awx/main/migrations/0185_move_JSONBlob_to_JSONField.py

```diff
@@ -0,0 +1,142 @@
+# Generated by Django 4.2 on 2023-06-09 19:51
+
+import awx.main.models.notifications
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('main', '0184_django_indexes'),
+        ('conf', '0010_change_to_JSONField'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='activitystream',
+            name='deleted_actor',
+            field=models.JSONField(null=True),
```

Comment:
Did this conversion not require a data migration? You could migrate locally and check the functionality of a single field and I would be satisfied. Right now I just don't see that documented anywhere (you may have told me and I forgot), so I want to see it written down.

Reply:
From sqlmigrate:
```
--
-- Alter field survey_spec on jobtemplate
--
ALTER TABLE "main_jobtemplate" ALTER COLUMN "survey_spec" TYPE jsonb USING "survey_spec"::jsonb;
```

Pre migration:
```
awx=# select unifiedjobtemplate_ptr_id, survey_spec from main_jobtemplate where unifiedjobtemplate_ptr_id = 7;
 unifiedjobtemplate_ptr_id |                                                                                                                                     survey_spec                                                                      >
---------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------->
                         7 | {"name": "", "spec": [{"max": 1024, "min": 0, "type": "multiplechoice", "choices": ["production", "development"], "default": "", "required": true, "variable": "environment", "new_question": true, "question_name": >
(1 row)
```

Post migration:
```
awx=# select unifiedjobtemplate_ptr_id, survey_spec from main_jobtemplate where unifiedjobtemplate_ptr_id = 7;
 unifiedjobtemplate_ptr_id |                                                                                                                                     survey_spec                                                                                     >
---------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------->
                         7 | {"name": "", "spec": [{"max": 1024, "min": 0, "type": "multiplechoice", "choices": ["production", "development"], "default": "", "required": true, "variable": "environment", "new_question": true, "question_name": "Environment", >
(1 row)
```

So the data "looks" the same but it should now be a jsonb instead of a text field so we can do "nifty" JSON selects:
```
select unifiedjobtemplate_ptr_id, survey_spec->>'spec' from main_jobtemplate WHERE survey_spec->>'name' = 'Get the zrelease';
 unifiedjobtemplate_ptr_id |                                                                                                    ?column?                                                                                                    
---------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
                        62 | [{"type": "text", "required": true, "variable": "release_version", "question_name": "What is the zrelease to branch?", "question_description": "Enter the version to zrelease in the format <major>.<minor>"}]
(1 row)
```